test/integration/targets/apt/tasks/url-with-deps.yml

```diff
@@ -0,0 +1,64 @@
+- name: Install https transport for apt
+  apt:
+    name: apt-transport-https
+    state: latest
+    force: yes
```

Comment:
Why specify state and force? Isn't the default of present without force sufficient in this case?